server/src/routes/v1/apps/handlers/createApp.js

```diff
@@ -134,9 +135,20 @@ module.exports = {
                 )
             } else {
                 organisation = organisations[0]
-                if (currentUserId !== organisation.created_by_user_id) {
-                    //should we allow anyone to create an app for an existing organisation?
-                    //throw Boom.unauthorized()
+
+                const org = await OrganisationService.findOne(
```

Comment:
Any reason to not use the `hasMember()`-function?

Reply:
Not really, copied from organisation handlers where it's done that way, but there we also use other properties which makes more sense to fetch the entire org. Replaced with hasUser

Reply:
If you need the org I guess it makes sense with the original way, so you don't need 2 database requests.